### PR TITLE
[netgen] no abolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
Have been a long time since my last absolute paths PR 😆

### CODE REVIEW

File: ports/netgen/portfile.cmake [hunk 1]
```diff
@@ -102,4 +102,7 @@ file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
 if("python" IN_LIST FEATURES)
   vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/${PYTHON3_SITE}/netgen/config.py" "CMAKE_INSTALL_PREFIX  = \"${CURRENT_PACKAGES_DIR}" "CMAKE_INSTALL_PREFIX_NOT_USABLE = \"")
+  if(NOT VCPKG_BUILD_TYPE)
```
Comment (Neumann-A) [inline ports/netgen/portfile.cmake:105]: I think you can delete the folder instead. debug netgen python bindings do not make sense with the current way vcpkg does stuff with python

**Comment by WangWeiLin-MV** [comment]
Would you mind adding issue case, or effect of the fix?

File: ports/netgen/portfile.cmake (hunk 1 above)
Comment (autoantwort) [reply ports/netgen/portfile.cmake:105]: Then I get
```
warning: Mismatching number of debug and release binaries.
Found 2 debug binaries:

  /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx/debug/lib/libnglib.a
  /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx/debug/lib/libngcore.a

Found 4 release binaries:

  /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx/lib/python3.11/site-packages/netgen/libngpy.so
  /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx/lib/python3.11/site-packages/pyngcore/pyngcore.cpython-311-darwin.so
  /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx/lib/libnglib.a
  /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx/lib/libngcore.a
```
but I can disable this check

Comment (autoantwort) [reply ports/netgen/portfile.cmake:105]: Hm removing the folder is pain, I think I keep the current implementation if that is okay?

**Comment by autoantwort** [comment]
> Would you mind adding issue case, or effect of the fix?

```
warning: There should be no absolute paths, such as the following, in an installed package:
  /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed
  /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/netgen
  /Users/leanderSchulten/git_projekte/vcpkg/downloads
Absolute paths were found in the following files:
  /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx/debug/lib/python3.11/site-packages/netgen/config.py
error: Found 1 post-build check problem(s). To submit these ports to curated catalogs, please first correct the portfile: /Users/leanderSchulten/git_projekte/vcpkg/ports/netgen/portfile.cmake
```

**Comment by WangWeiLin-MV** [comment]
> ```
> warning: There should be no absolute paths, such as the following, in an installed package:
>   /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx
>   /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed
>   /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/netgen
>   /Users/leanderSchulten/git_projekte/vcpkg/downloads
> Absolute paths were found in the following files:
>   /Users/leanderSchulten/git_projekte/vcpkg/packages/netgen_arm64-osx/debug/lib/python3.11/site-packages/netgen/config.py
> error: Found 1 post-build check problem(s). To submit these ports to curated catalogs, please first correct the portfile: /Users/leanderSchulten/git_projekte/vcpkg/ports/netgen/portfile.cmake
> ```

Sorry there was no failure in my local `vcpkg install` and this log was not found in `buildtress`. And my local test building failed with not `occ_libs`.

Could you please give some reproduce steps and a simple example case?

**Comment by autoantwort** [comment]
`vcpkg x-set-installed 'netgen[python]'`

**Review by WangWeiLin-MV — APPROVED** [review]
The port installation tests pass with the following triplets:
* arm64-osx
* x64-linux

**Pull request merged**